creativecloud/blocks/callout/callout.js

```diff
@@ -34,8 +34,7 @@ const groupElementsIntoBlocks = (container, createTag) => {
 
 export default async function init(el) {
   const miloLibs = getLibs('/libs');
-  const { createTag, getConfig, loadStyle } = await import(`${miloLibs}/utils/utils.js`);
-  loadStyle(`${getConfig().base}/blocks/callout/callout.css`);
+  const { createTag } = await import(`${miloLibs}/utils/utils.js`);
```

Comment:
@Axelcureno where is the hardcoded path mentionned in the PR title?

Reply:
I have updated the title to reflect the actual change.